src/robot/conf/languages.py

```diff
@@ -170,3 +170,265 @@ class Fi(Language):
+    library = 'Bibliotheek'
+    resource = 'Resource'
+    variables = 'Variabele'
+    documentation = 'Documentatie'
+    metadata = 'Metadata'
+    suite_setup = 'Suite Preconditie'
+    suite_teardown = 'Suite Postconditie'
+    test_setup = 'Test Preconditie'
+    test_teardown = 'Test Postconditie'
+    test_template = 'Test Sjabloon'
+    test_timeout = 'Test Time-out'
+    force_tags = 'Forceer Labels'
+    default_tags = 'Standaard Labels'
+    tags = 'Labels'
+    setup = 'Preconditie'
+    teardown = 'Postconditie'
+    template = 'Sjabloon'
+    timeout = 'Time-out'
+    arguments = 'Parameters'
+    return_ = 'Return'
+    bdd_prefixes = {'Stel', 'Als', 'Dan', 'En', 'Maar'}
+
+class PL(Language):
+    setting_headers = {'Ustawienia', 'Ustawienia', 'Ustawienia', 'Ustawienia'}
+    variable_headers = {'Zmienna', 'Zmienne', 'Zmienne', 'Zmienne'}
+    test_case_headers = {'Test Case', 'Test Cases', 'Test Cases', 'Test Cases'}
+    task_headers = {'Zadanie', 'Zadania', 'Zadania', 'Zadania'}
+    keyword_headers = {'Keyword', 'Keywords', 'Keywords', 'Keywords'}
+    comment_headers = {'Comment', 'Comments', 'Comments', 'Comments'}
+    library = 'Biblioteka'
+    resource = 'Zasób'
+    variables = 'Zmienna'
+    documentation = 'Dokumentacja'
+    metadata = 'Metadane'
+    suite_setup = 'Suite Setup'
+    suite_teardown = 'Suite Teardown'
+    test_setup = 'Test Setup'
+    test_teardown = 'Test Teardown'
+    test_template = 'Test Template'
+    test_timeout = 'Limit Czasu Testu'
+    force_tags = 'Wymuś Znaczniki'
+    default_tags = 'Domyślne Znaczniki'
+    tags = 'Znaczniki'
+    setup = 'Ustawienia'
+    teardown = 'Teardown'
+    template = 'Szablon'
+    timeout = 'Limit Czasu'
+    arguments = 'Argumenty'
+    return_ = 'Return'
+    bdd_prefixes = {'Given', 'Kiedy', 'Wtedy', 'Oraz', 'Ale'}
+
+class PT(Language):
+    setting_headers = {'Definição', 'Definições'}
+    variable_headers = {'Variável', 'Variáveis'}
+    test_case_headers = {'Caso de Teste', 'Casos de Teste'}
+    task_headers = {'Tarefa', 'Tarefas'}
+    keyword_headers = {'Palavra-Chave', 'Palavras-Chave'}
+    comment_headers = {'Comentário', 'Comentários'}
+    library = 'Biblioteca'
+    resource = 'Recurso'
+    variables = 'Variável'
+    documentation = 'Documentação'
+    metadata = 'Metadados'
+    suite_setup = 'Inicialização de Suíte'
+    suite_teardown = 'Finalização de Suíte'
+    test_setup = 'Inicialização de Teste'
+    test_teardown = 'Finalização de Teste'
+    test_template = 'Modelo de Teste'
+    test_timeout = 'Tempo Limite de Teste'
+    force_tags = 'Forçar Etiquetas'
+    default_tags = 'Etiquetas por Defeito'
+    tags = 'Etiquetas'
+    setup = 'Inicialização'
+    teardown = 'Finalização'
+    template = 'Modelo'
+    timeout = 'Tempo Limite'
+    arguments = 'Argumentos'
+    return_ = 'Retornar'
+    bdd_prefixes = {'Dado', 'Quando', 'Então', 'E', 'Mas'}
+
+from robot.conf import Language
+class PT-br(Language):
```

Comment:
The import isn't needed and the `PT-br` class name is a syntax error. It should be changed to `PtBr`. Probably should update the script that generates this Python code as well.

Reply:
I opted for Pt-Br. Should that still be changed to PtBr or is the code updated?